v63002/outref/gtm8766.txt

```diff
@@ -1,7 +1,7 @@
-# In the current version, 2MB is the maximum value accepted by both MUPIP and GDE. In previous versions
-# 2MB was the max for MUPIP, but a core would be produced for a value greater than or equal to 2GB, and 2GB was the max value for GDE
+##TEST_AWK# In the current version, [0-9]* is the maximum value accepted by both MUPIP and GDE. In previous versions
```

Comment:
I would like (2Mb|64Kb) syntax instead of [0-9]*. That would ensure the only allowed choices are two instead of anything.